data/transactions/logic/opcodes.go

```diff
@@ -249,6 +250,7 @@ var OpSpecs = []OpSpec{
 	{0x59, "extract_uint16", opExtract16Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
 	{0x5a, "extract_uint32", opExtract32Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
 	{0x5b, "extract_uint64", opExtract64Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
+	{0x5c, "base64_decode", opBase64Decode, assembleBase64Decode, disBase64Decode, oneBytes, oneBytes, 6, modeAny, costlyImm(25, "v")},
```

Comment:
I'm arguing for a cost of 25 because of [this analysis](https://docs.google.com/spreadsheets/d/18y-rZUqj3QeZKtMnlPFR8q66f1_B5phdb2jOwwPQfQA/edit#gid=1096786921), but it is up for discussion. In particular, the benchmark data than ran on my laptop shows that for 1 KiB base64 encoded data:

* if we compare with `b~`, the cost should be around 6
* if we compare with `sha512_256`, the cost should be around 20
* if we compare wtih `keccak256`, the cost should be around 33

So a conservative cost estimate stands at 25.